tools/istio-docker.mk

```diff
@@ -126,15 +126,12 @@ docker.pilot: BUILD_PRE=; chmod 755 pilot-discovery cacert.pem
 docker.pilot: BUILD_ARGS=--build-arg BASE_VERSION=${BASE_VERSION}
 docker.pilot: $(ISTIO_OUT_LINUX)/pilot-discovery
 docker.pilot: tests/testdata/certs/cacert.pem
+docker.pilot: sidecar-injector/inject/config
+docker.pilot: sidecar-injector/inject/values
+docker.pilot: pilot/docker/mesh
```

Comment:
This doesn't actually do anything if they aren't added in the Dockerfile

Reply:
Ops, that's the next PR. Tricky to keep them separated.